src/Annotations/Serializer/CommonMark/functions.php

```diff
@@ -2,9 +2,25 @@
 
 namespace eLife\Annotations\Serializer\CommonMark;
 
-function clean_paragraph($text)
+use League\CommonMark\Util\Xml;
+
+function clean_paragraph(string $text) : string
 {
     $allowed_tags = '<i><sub><sup><span><del><math><a><br><caption>';
```

Comment:
What's the difference between `<math>` here and `escape_math()`?

Is it a requirement to drop MathML? Unlike LaTeX we can display it...

Reply:
Not a requirement. I think it will not be too much trouble do capture and display it particularly when it's in a block rather than inline. But I will take that on as a followup to this. `<math>` was left in while I was deciding how to treat mathml. I will remove it for now while we are simply escaping MathML.